src/GraphQL.SystemTextJson/StringExtensions.cs

```diff
@@ -1,123 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Numerics;
-using System.Text.Json;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace GraphQL.SystemTextJson
-{
-    /// <summary>
-    /// Provides extension methods to deserialize json strings into object dictionaries.
-    /// </summary>
-    public static class StringExtensions
-    {
-        private static readonly JsonSerializerOptions _jsonOptions = new JsonSerializerOptions
-        {
-            PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
-            Converters =
-            {
-                new InputsConverter(),
-                new JsonConverterBigInteger(),
-            }
-        };
-
-        private static readonly JsonSerializerOptions _jsonOptions2 = new JsonSerializerOptions
-        {
-            PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
-            Converters =
-            {
-                new InputsConverter(),
-                new JsonConverterBigInteger(),
-            }
-        };
-
-        /// <summary>
-        /// Converts a JSON-formatted string into a dictionary.
-        /// </summary>
-        /// <param name="json">A JSON formatted string.</param>
-        /// <returns>Inputs.</returns>
-        public static Inputs ToInputs(this string json)
-            => json != null ? JsonSerializer.Deserialize<Inputs>(json, _jsonOptions2) : Inputs.Empty;
-
-        /// <summary>
-        /// Deserializes a JSON-formatted string of data into the specified type.
-        /// <br/><br/>
-        /// Any <see cref="Inputs"/> objects will be deserialized into the proper format.
-        /// <br/><br/>
-        /// Property names are converted to camel case and matched based on a case sensitive comparison (the default for System.Text.Json).
-        /// </summary>
-        public static T FromJson<T>(this string json)
-            => JsonSerializer.Deserialize<T>(json, _jsonOptions2);
-
-        /// <summary>
-        /// Deserializes a JSON-formatted stream of data into the specified type.
-        /// <br/><br/>
-        /// Any <see cref="Inputs"/> objects will be deserialized into the proper format.
-        /// <br/><br/>
-        /// Property names are converted to camel case and matched based on a case sensitive comparison (the default for System.Text.Json).
-        /// </summary>
-        public static ValueTask<T> FromJsonAsync<T>(this System.IO.Stream stream, CancellationToken cancellationToken = default)
-            => JsonSerializer.DeserializeAsync<T>(stream, _jsonOptions2, cancellationToken);
-
-        /// <summary>
-        /// Converts a JSON element into an <see cref="Inputs"/> object.
-        /// </summary>
-        public static Inputs ToInputs(this JsonElement obj)
```

Comment:
Where is all that code from `ToInputs`? If removed then why did we use it before?

Reply:
It was written before `InputsConverter` existed.  There is no need for it as the `InputsConverter` does the same thing.

Reply:
Correction: `Payload` deserializes to `GraphQLRequest` typically.  Not sure what the use case is for `ToInptus` on a `JsonElement`.